mgmtsystem/virtualcenter.py

```diff
@@ -445,21 +448,34 @@ def in_steady_state(self, vm_name):
         return self.vm_status(vm_name) in {self.POWERED_ON, self.POWERED_OFF, self.SUSPENDED}
 
     def is_vm_running(self, vm_name):
-        return self.vm_status(vm_name) == self.POWERED_ON
+        status = self.vm_status(vm_name)
```

Comment:
This is a bit redundant I guess, it'll still be false when `None == self.STATE`.